graphene_django/filter/fields.py

```diff
@@ -59,7 +61,13 @@ def resolve_queryset(
             connection, iterable, info, args
         )
         filter_kwargs = {k: v for k, v in args.items() if k in filtering_args}
-        return filterset_class(data=filter_kwargs, queryset=qs, request=info.context).qs
+        filterset = filterset_class(
+            data=filter_kwargs, queryset=qs, request=info.context
+        )
+        if filterset.is_valid():
+            return filterset.qs
+        else:
+            raise ValidationError(json.dumps(filterset.errors.get_json_data()))
```

Comment:
```suggestion
        raise ValidationError(json.dumps(filterset.errors.get_json_data()))
```

No need for the `else` statement.